telegram/_stars.py

```diff
@@ -387,31 +416,31 @@ def de_json(cls, data: Optional[JSONDict], bot: "Bot") -> Optional["StarTransact
         if not data:
             return None
 
-        if "date" in data:
-            # Get the local timezone from the bot if it has defaults
-            loc_tzinfo = extract_tzinfo_from_defaults(bot)
-            data["date"] = from_timestamp(data["date"], tzinfo=loc_tzinfo)
+        # Get the local timezone from the bot if it has defaults
+        loc_tzinfo = extract_tzinfo_from_defaults(bot)
+        data["date"] = from_timestamp(data.get("date", None), tzinfo=loc_tzinfo)
 
         data["source"] = TransactionPartner.de_json(data.get("source"), bot)
+        print(data.get("receiver"))
```

Comment:
🙃

Reply:
aghh, I'm going to tell ruff to fix that